src/sentry/search/snuba/executors.py

```diff
@@ -453,6 +461,24 @@ def trend_aggregation(start: datetime, end: datetime) -> Sequence[str]:
     ]
 
 
+def better_priority_aggregation(
+    start: datetime, end: datetime, age: int, log_level: int, frequency: int, has_stacktrace: int
```

Comment:
I assume `age`, `log_level`, etc etc are just here for future use?

I think these should maybe be like `<name>_weight` since at the moment they read like they're the actual values.

Or maybe they should all be members of some `weight` dictionary? That way you keep the namespace clear for other things like `event_score_halflife` etc.

We should probably also exclude `age` here. It's useful if we're using the weight scores alone, but doesn't make as much sense when using time decay.